examples/complete/android/demo/src/main/res/values/strings.xml

```diff
@@ -90,7 +90,7 @@
         "For example, a floating action button (FAB) can disappear or move off screen before a "
         "user picks up a card, or it can move if a snackbar appears.\n"
         "On the layout level, design your app layout to minimize opportunities for interference. "
-        "For example, position the FAB to one side of stream of a cards so the FAB won’t interfere "
+        "For example, position the FAB to one side of stream of a cards thus the FAB won’t interfere "
```

Comment:
For example, position the FAB to one side of stream of a cards so the FAB won’t interfere 

I realize this one is putting 'so' back in - I think this is an appropriate use.

Reply:
:+1: